python/lsst/obs/base/gen3/ingest.py

```diff
@@ -353,87 +322,31 @@ def extractDataId(self, file, headers):
 
         Returns
         -------
-        dataId : `dict`
-            Must include "instrument", "detector", and "exposure" keys. If the
-            Exposure is associated with a PhysicalFilter and/or Visit,
-            "physical_filter" and "visit" keys should be provided as well
-            (respectively).
+        dataId : `DataId`
+            A mapping whose key-value pairs uniquely identify raw datasets.
+            Must have ``dimensions`` equal to ``self.dimensions``.
```

Comment:
If `dataId.dimensions` is equal to `self.dimensions`, shouldn't the removed dimensions (`Visit`, `PhysicalFilter`) below already be known?